### PR TITLE
Selenium

### PR DESCRIPTION
**Pull request by Neitsch**
Introducing selenium testing

### CODE REVIEW

**Review by infobiac — APPROVED** [review]
Don’t tell me what I can or can’t review

File: authentication/plaid_middleware.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 PLAID_CLIENT_ID = os.environ.get('PLAID_CLIENT_ID')
 PLAID_SECRET = os.environ.get('PLAID_SECRET')
 PLAID_PUBLIC_KEY = os.environ.get('PLAID_PUBLIC_KEY')
-PLAID_ENV = 'sandbox' if os.environ.get('DEBUG') == "TRUE" else 'development'
+PLAID_ENV = 'sandbox'
```
Comment (infobiac) [inline authentication/plaid_middleware.py]: why you do dis

File: BuyBitcoin/templates/setup_bank.html [hunk 2]
```diff
@@ -5,7 +5,8 @@
 <script src="https://cdn.plaid.com/link/v2/stable/link-initialize.js"></script>
 
 <button id="link-button">Open Link</button>
-
+<form>{% csrf_token %}
```
Comment (infobiac) [inline BuyBitcoin/templates/setup_bank.html]: what's this for

File: BuyBitcoin/settings.py [hunk 3]
```diff
@@ -169,7 +173,8 @@
 
 # Simplified static file serving.
 # https://warehouse.python.org/project/whitenoise/
-STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+# STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+WHITENOISE_ROOT = os.path.join(BASE_DIR, 'assets')
```
Comment (infobiac) [inline BuyBitcoin/settings.py:177]: does this still access middleware properly?

File: authentication/plaid_middleware.py (hunk 1 above)
Comment (Neitsch) [reply authentication/plaid_middleware.py]: Cuz Travis doesn't set DEBUG to true. Going to use a separate env variable.

File: BuyBitcoin/templates/setup_bank.html (hunk 2 above)
Comment (Neitsch) [reply BuyBitcoin/templates/setup_bank.html]: CSRF token was missing. This makes sure that it is present for the plaid token sending

File: BuyBitcoin/settings.py (hunk 3 above)
Comment (Neitsch) [reply BuyBitcoin/settings.py:177]: How do I verify that?

File: BuyBitcoin/templates/setup_bank.html (hunk 2 above)
Comment (infobiac) [reply BuyBitcoin/templates/setup_bank.html]: Doesn't line 25 have the csrf token?

File: BuyBitcoin/settings.py (hunk 3 above)
Comment (infobiac) [reply BuyBitcoin/settings.py:177]: I think if you try python manage.py collectstatic and it exits without complaints you're good.

**Review by infobiac — APPROVED** [review]

File: BuyBitcoin/templates/setup_bank.html (hunk 2 above)
Comment (Neitsch) [reply BuyBitcoin/templates/setup_bank.html]: No, we need to make sure that the CSRF token is provided as a cookie. Line 25 only grabs it once it is present.

**Pull request merged**